.github/workflows/bleeding-edge.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow installs dependencies from master
+
+name: ETS from source
+
+# TODO: Switch to schedule!
+on: push
+
+jobs:
+  test-ets:
+    strategy:
+      matrix:
+        # Note that Ubuntu 20 cannot be used because its SWIG version is 4.0
+        # (enthought/enable#360)
+        os: [ubuntu-18.04]
+        toolkit: ['pyside2', 'wx']
+        python-version: [3.8]
```

Comment:
I tried the combination of Python 3.9 and Ubuntu 20, but there aren't a prebuilt wheel for wxPython, which means we will have to build from source (take a long time!). But then I haven't tried bumping Python version again for Ubuntu 18.

I did not use Ubuntu 16, as an attempt to be as close to the latest as possible, and also to be different from the per-PR CI job (using Ubuntu 16) to expand coverage.